app/routers/game_release_dates_service.py

```diff
@@ -0,0 +1,170 @@
+import datetime
+
+from fastapi import APIRouter, Depends, Request
+from fastapi.responses import RedirectResponse, Response
+from sqlalchemy.orm import Session
+from starlette.status import HTTP_302_FOUND
+
+import requests
+from app.database.models import UserSettings
+from app.dependencies import get_db, templates
+from app.internal.security.dependencies import current_user
+from app.internal.security.schema import CurrentUser
+from app.internal.utils import create_model
+
+from typing import Dict, List
+
+
+router = APIRouter(
+    prefix="/game-releases",
+    tags=["game-releases"],
+    responses={404: {"description": "Not found"}},
+)
+
+
+def is_user_signed_up_for_game_releases(
+    session: Session,
+    current_user_id: int,
+) -> bool:
+    is_signed_up = (
+        session.query(UserSettings)
+        .filter(UserSettings.user_id == current_user_id)
+        .filter(UserSettings.video_game_releases.is_(True))
+        .first()
+    )
+
+    if is_signed_up:
+        return True
+    return False
```

Comment:
`return is_signed_up`

Reply:
is_signed_up is an instance not a bool when it exists